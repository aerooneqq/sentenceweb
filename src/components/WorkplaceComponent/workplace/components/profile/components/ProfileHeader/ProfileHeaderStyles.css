#profileHeader { 
    font-family: Arial; 
    font-size: 30px !important; 
    color: orangered;
    cursor: default;

    margin-bottom: 5px;

    text-align: left;
}