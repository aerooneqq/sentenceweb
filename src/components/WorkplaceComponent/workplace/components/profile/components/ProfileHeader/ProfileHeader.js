import React, {Component} from "react"

//Styles
import "./ProfileHeaderStyles.css"

/**
 * PROPS LIST:
 * 1) header - the name of the header to be displayed
 */
export default class ProfileHeader extends Component { 
    render() { 
        return ( 
            <div id="profileHeader">
                {this.props.header}
            </div>
        )
    }
}