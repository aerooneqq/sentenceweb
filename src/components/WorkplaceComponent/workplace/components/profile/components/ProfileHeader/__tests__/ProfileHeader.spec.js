import React from "react";
import {shallow} from "enzyme"; 
import {shallowToJson} from "enzyme-to-json";
import ProfileHeader from "../ProfileHeader";

describe("ProfileHeader", () => { 
    it("should render correctly", () => { 
        expect(shallowToJson(shallow(<ProfileHeader />))).toMatchSnapshot();
    });

    it("should correctly display name of the header", () => { 
        const props = { 
            header: "test name",
        }

        const profileHeader = shallow(<ProfileHeader {...props} />);
        expect(profileHeader.find("div").text()).toBe(props.header);
    })
});