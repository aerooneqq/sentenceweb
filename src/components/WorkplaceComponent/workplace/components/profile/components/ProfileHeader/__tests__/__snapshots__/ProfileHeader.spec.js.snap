// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileHeader should render correctly 1`] = `
<div
  id="profileHeader"
/>
`;
